core/jobs/src/main/java/com/cognifide/aet/job/common/collectors/source/SourceCollectorFactory.java

```diff
@@ -20,25 +20,19 @@
 import com.cognifide.aet.job.api.collector.CollectorProperties;
 import com.cognifide.aet.job.api.collector.WebCommunicationWrapper;
 import com.cognifide.aet.job.api.exceptions.ParametersException;
+import com.cognifide.aet.job.common.collectors.source.configuration.SourceCollectorFactoryConf;
 import com.cognifide.aet.vs.ArtifactsDAO;
 import java.util.Map;
-import org.apache.felix.scr.annotations.Activate;
-import org.apache.felix.scr.annotations.Component;
-import org.apache.felix.scr.annotations.Property;
-import org.apache.felix.scr.annotations.Reference;
-import org.apache.felix.scr.annotations.Service;
-import org.apache.sling.commons.osgi.PropertiesUtil;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.metatype.annotations.Designate;
 
-@Component(metatype = true, description = "AET Source Collector Factory", label = "AET Source Collector Factory")
-@Service
+@Component(service = CollectorFactory.class)
```

Comment:
`(service = CollectorFactory.class)` is redundant - type of directly implemented interface is the default service value